### PR TITLE
Fix #140. Refactor the  offerings section

### LINKED ISSUE #140: Remove styled components from the offerings section
Currently the offerings component uses still styled components https://github.com/hardforkio/homepage/blob/974363c113eac7c66fb634f89e21aa05b8a79cd5/src/components/Offerings/index.jsx#L8
This should be removed.

### PR DESCRIPTION
**Pull request by kiliw**
Fix #140.
Remove styled components.

### CODE REVIEW

File: src/components/Offerings/index.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  offerings,
+}) => (
+  <div>
+    {R.addIndex<Offering>(R.map)((item, index) => (
+      <OfferingComponent
+        index={index}
+        image={item.image}
+        headline={item.headline}
+        text={item.text}
+      />
```
Comment (Levino) [inline src/components/Offerings/index.tsx]: Move this to helper function "mapToComponent"

Comment (kiliw) [reply src/components/Offerings/index.tsx]: Done

File: src/utils/helpers.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import * as R from 'ramda'
+import { FunctionComponent } from 'react'
+
+export const mapToComponent = (Component: FunctionComponent<any>, list: any) =>
```
Comment (Bakkhos) [inline src/utils/helpers.tsx]:
```suggestion
export const mapToComponent = <T>(Component: FunctionComponent<T>, props: T[]) =>
```

Comment (kiliw) [reply src/utils/helpers.tsx]: Thanks, added it.

**Pull request merged**